_posts/2025-06-26-hibernate7-on-quarkus.adoc

```diff
@@ -93,7 +93,7 @@ var books =
 
 === Hibernate Reactive together with Hibernate ORM
 
-A long-awaited feature is the ability to https://github.com/quarkusio/quarkus/issues/13425[mix Hibernate ORM and Hibernate Reactive extensions] in the same Quarkus application. Without this feature, making the two extensions coexist required workarounds that are now unnecessary as, since Quarkus 3.2.4, it's now possible to mix the two.
+A long-awaited feature is the ability to https://github.com/quarkusio/quarkus/issues/13425[mix Hibernate ORM and Hibernate Reactive extensions] in the same Quarkus application. Without this feature, making the two extensions coexist required workarounds that are now unnecessary: since Quarkus 3.2.4, it's now possible to mix the two.
```

Comment:
I know it was preexisting but it needs a fix!

```suggestion
A long-awaited feature is the ability to https://github.com/quarkusio/quarkus/issues/13425[mix Hibernate ORM and Hibernate Reactive extensions] in the same Quarkus application. Without this feature, making the two extensions coexist required workarounds that are now unnecessary: since Quarkus 3.24, it's now possible to mix the two.
```